dnstable_convert.c

```diff
@@ -1012,12 +1022,14 @@ init_mtbl(mtbl_compression_type compression, int level, size_t mem_mb)
 static void
 usage(const char *name)
 {
-	fprintf(stderr, "Usage: %s [-D] [-p] [-r] [-S] [-c compression] [-l level] [-m megabytes] [-s NAME] <NMSG FILE> <DB FILE> <DB DNSSEC FILE>\n", name);
+	fprintf(stderr, "Usage: %s [-D] [-p] [-r] [-S] [-c compression] [-l level] [-b size] [-t threads] [-m megabytes] [-s NAME] <NMSG FILE> <DB FILE> <DB DNSSEC FILE>\n", name);
```

Comment:
This requires a corresponding man page entry.

Another nitpick: note the almost perfect pre-existing alphabetical ordering convention in the getopt() string itself, the subsequent option switch(), and the usage output.